grafana-plugin/src/assets/style/utils.css

```diff
@@ -117,6 +117,10 @@
  * Icons
  */
 
+.loader {
+  margin-bottom: 0 !important;
```

Comment:
Could you explain why do we need !important? Where we have a problem with precedence?

Reply:
So that it overwrites styling from grafana